std/experimental/allocator/building_blocks/kernighan_ritchie.d

```diff
@@ -111,34 +111,38 @@ struct KRRegion(ParentAllocator = NullAllocator)
 
         this(this) @disable;
 
+        nothrow @nogc @safe
         void[] payload() inout
         {
-            return (cast(ubyte*) &this)[0 .. size];
+            return (() @trusted => (cast(ubyte*) &this)[0 .. size])();
         }
 
+        nothrow @nogc @safe
         bool adjacent(in Node* right) const
         {
             assert(right);
             auto p = payload;
-            return p.ptr < right && right < p.ptr + p.length + Node.sizeof;
+            return p.ptr < right && right < (() @trusted => (p.ptr + p.length + Node.sizeof))();
         }
 
+        nothrow @nogc @safe
         bool coalesce(void* memoryEnd = null)
         {
             // Coalesce the last node before the memory end with any possible gap
             if (memoryEnd
-                && memoryEnd < payload.ptr + payload.length + Node.sizeof)
+                && memoryEnd < (() @trusted => (payload.ptr + payload.length + Node.sizeof))())
```

Comment:
Probably same thing here, you're adding 3 lambdas for a relatively small function, it's not worth it IMO.